libfuse/folderlist.go

```diff
@@ -24,6 +25,9 @@ type FolderList struct {
 
 	mu      sync.Mutex
 	folders map[string]*TLF
+
+	recentlyRemoved   map[libkbfs.CanonicalTlfName]bool
+	muRecentlyRemoved sync.RWMutex
```

Comment:
We usually put the mutex above the fields it protects (the recommended style for golang).